java-symbol-solver-core/src/main/java/me/tomassetti/symbolsolver/javaparsermodel/JavaParserFacade.java

```diff
@@ -531,6 +527,14 @@ private Type getTypeConcrete(Node node, boolean solveLambdas) {
         }
     }
 
+    public Type convertToUsageVariableType(VariableDeclarationExpr var) {
+        Type type = JavaParserFacade.get(typeSolver).convertToUsage(var.getElementType(), var);
+        for (int i=0;i<var.getArrayBracketPairsAfterElementType().size();i++) {
+            type = new ArrayType(type);
```

Comment:
Double-check the last post here: https://github.com/javaparser/javaparser/issues/237
That details all the changes :)
And thanks for supplying feedback!